src/seq.rs

```diff
@@ -457,6 +474,22 @@ bitflags! {
     }
 }
 
+bitflags! {
+    /// [SND_SEQ_REMOVE_xxx]https://www.alsa-project.org/alsa-doc/alsa-lib/group___seq_event.html) constants 
+    pub struct RemoveEventsCondition: u32 {
```

Comment:
Rename to "Remove"

Reply:
Renamed.  (Not sure what etiquette is for these change requests... I'll let you resolve the conversations)